codespell_lib/data/dictionary.txt

```diff
@@ -4688,7 +4722,9 @@ behaviuor->behaviour
 behavoir->behavior
 behavoirs->behaviors
 behavor->behavior, behaviour,
+behavoral->behavioral
```

Comment:
`behavoral->behavioral, behavioural,` for consistency with the previous line?

Reply:
Makes sense, yes. Added.